tests/select_for_update/tests.py

```diff
@@ -179,6 +179,20 @@ def test_for_update_requires_transaction_only_in_execution(self):
         with self.assertRaises(transaction.TransactionManagementError):
             list(people)
 
+    @skipUnlessDBFeature('supports_select_for_update_with_limit')
+    def test_select_for_update_with_limit(self):
+        other = Person.objects.create(name='Other')
+        with transaction.atomic():
+            person = Person.objects.all().order_by('pk').select_for_update()[1:2].get()
```

Comment:
`get()` adds redundant noise here IMO (outside of Oracle, `get()` adds its own slicing IIRC). I'd prefer:
```
    qset = Person.objects.all().order_by('pk').select_for_update()[1:2]
    person = list(qset)[0]
```